crates/uv/tests/it/init.rs

```diff
@@ -1954,6 +1986,14 @@ fn init_nested_virtual_workspace() -> Result<()> {
             workspace, @r###"
         [tool.uv.workspace]
         members = ["foo"]
+
+        [tool.uv.sources]
+        foo = { workspace = true }
+
+        [project]
+        dependencies = [
+            "foo",
+        ]
```

Comment:
Hm this is probably wrong, we shouldn't create a project table / add it in a virtual workspace.